plugins/OpenDeck/OpenDeckPlugin.cpp

```diff
@@ -0,0 +1,96 @@
+/* Copyright (c) 2018, EPFL/Blue Brain Project
+ * All rights reserved. Do not distribute without permission.
+ *
+ * This file is part of Brayns <https://github.com/BlueBrain/Brayns>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include "OpenDeckPlugin.h"
+
+#include <brayns/common/camera/Camera.h>
+#include <brayns/common/engine/Engine.h>
+#include <brayns/common/log.h>
+#include <brayns/pluginapi/PluginAPI.h>
+
+#include <ospray/SDK/camera/PerspectiveCamera.h>
+
+namespace brayns
+{
+namespace
+{
+constexpr uint32_t openDeckWallResX = 11940u;
+constexpr uint32_t openDeckWallResY = 3424u;
+constexpr uint32_t openDeckFloorResX = 4096u;
+constexpr uint32_t openDeckFloorResY = 2125u;
+constexpr char leftWallBufferName[] = "0L";
+constexpr char rightWallBufferName[] = "0R";
+constexpr char leftFloorBufferName[] = "1L";
+constexpr char rightFloorBufferName[] = "1R";
+}
+
+OpenDeckPlugin::OpenDeckPlugin(const Vector2ui& wallRes,
+                               const Vector2ui& floorRes)
+    : _wallRes(wallRes)
+    , _floorRes(floorRes)
+{
+}
+
+void OpenDeckPlugin::init(PluginAPI* api)
+{
+    _api = api;
```

Comment:
that member is actually not needed. It should be once your plugin gets unloaded to remove the buffers again.

Reply:
Done